starter_code/src/movies.js

```diff
@@ -2,12 +2,73 @@
 // Turn duration of the movies from hours to minutes 
 
 
+function turnHoursToMinutes(moviesArr) {
+
+  return moviesArr.map(function(element) {
+
+    function stringToMinutes(string) {
+    var arr = string.split('h')
+    var hours = Number(arr[0])
+    var minutes = Number(arr[1].split("min")[0])
+    
+      return hours*60+minutes
+    } 
+  })
+}
+/* function stringToMinutes(string) {
+    var arr = string.split('h')    
+    if (arr.length < 2) {
+        arr.unshift("0")        
+    }    
+    var hours = Number(arr[0])
+    var minutes = Number(arr[1].split("min")[0])
+
+    return hours*60+minutes
+}
+
+*/ 
+
+
 // Get the average of all rates with 2 decimals 
 
+function ratesAverage (array) {
+
+var sum = array.reduce(function(accu, current) {
+
+return accu + parseFloat(current.rate)
+
+}, 0) 
+
+var count = array.length
+
+return sum / count;
+
+}
 
 // Get the average of Drama Movies
 
 
+function dramaMoviesRate (array) {
+
+var filter = array.filter(function(first){
+ return first.genre.includes("Drama")
```

Comment:
maybe `first` is not a good name for the parameter, since it does not only filter the first ;) `element` or `movie` would be better I think